tests/e2e/cosmos-integration-e2e/clientcontroller/babylon/babylon.go

```diff
@@ -157,10 +157,14 @@ func (bc *BabylonController) RegisterFinalityProvider(
 
 	fpAddr := bc.MustGetTxSigner()
 	msg := &btcstakingtypes.MsgCreateFinalityProvider{
-		Addr:        fpAddr,
-		BtcPk:       fpPk,
-		Pop:         &bbnPop,
-		Commission:  commission,
+		Addr:  fpAddr,
+		BtcPk: fpPk,
+		Pop:   &bbnPop,
+		Commission: btcstakingtypes.CommissionRates{
+			Rate:          *commission,
+			MaxRate:       math.LegacyMustNewDecFromStr("0.2"),
```

Comment:
Nit: Isn't there a shorter / handier form of `LegacyMustNewDecFromStr` in the SDK? Just curious.

Reply:
Hmm I did not find one tbh :sweat: